packages/data-models/constants.ts

```diff
@@ -165,5 +172,13 @@ const APP_CONSTANTS = {
   SERVER_SYNC_FREQUENCY_MS,
 };
 // Export as a clone to avoid risk one import could alter another
-export const getDefaultAppConstants = () => clone(APP_CONSTANTS);
-export type IAppConstants = typeof APP_CONSTANTS;
+export const getDefaultAppConfig = () => clone(APP_CONFIG);
+export type IAppConfig = typeof APP_CONFIG;
+
+/** A recursive version of Partial, making all properties, included nested ones, optional.
+ * Copied from https://stackoverflow.com/a/47914631
+ */
+type RecursivePartial<T> = {
```

Comment:
Very cool, never thought of defining a partial recursively but makes a lot of sense